src/main/webapp/js/form.js

```diff
@@ -1,7 +1,7 @@
 /* ==========================================================================
    SEARCH FORM
    ========================================================================== */
-$("#randomize-form").submit((event) => {
+$("#randomize-form").submit(function(event) {

```

Comment:
is this change necessary?

Reply:
Oh I see. We would have to replace `$(this)` usages with `event.currentTarget` if we want to use arrows.

Reply:
Yea the arrow function isn't working with the AJAX selector and it crashes the app

Reply:
Should I use the arrow function and change the $this or keep the current state?

Reply:
arrow function is generally preferred